.idea/.idea.osu.Desktop/.idea/discord.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why add files?

Reply:
Uh, idk, I've never added it tbh lmao, I don't know where does it come from

Reply:
It comes from my discord RPC, should i remove the plugin?

Reply:
you should not commit this. add it to your global .gitignore

Reply:
Okay, i will do this

Reply:
Done, i've added it to the .gitignore, i will remove it from my fork